ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java

```diff
@@ -37,9 +40,12 @@
   private static ReactFontManager sReactFontManagerInstance;
 
   private Map<String, FontFamily> mFontCache;
+  private Map<String, Typeface> mCustomTypefaceCache;
+  private boolean mHasCustomTypeface = false;
```

Comment:
Technically you could just make `mCustomTypefaceCache` `@Nullable`, then default to `null`. If someone adds a custom typeface, you initialize it as a new HashMap, then you don't need `mHasCustomTypeface` boolean. You can just check if `mCustomTypefaceCache != null`

Reply:
Yes, my first version was like that but changed it to use booleans just before the commit because it looked cleaner.

Reply:
from @mdvacca:

let's add final to these variables